projects/storefrontlib/cms-components/product/product-intro/product-intro.component.spec.ts

```diff
@@ -192,18 +189,6 @@ describe('ProductIntroComponent in product', () => {
       );
     });
 
-    it('should not display Show Reviews when reviews are not available', () => {
```

Comment:
These unit tests were removed because the tab component does not render the reviews component until it is clicked. The "Show Reviews" button behaviour was based on showing the button if the reviews component existed. It now checks whether there is an average reviews and renders the button based on this.

Note to QA: We should test the "Show Reviews" button on the product details page still works correctly based on changes made to fix [this ticket](https://github.com/SAP/spartacus/issues/14030).